tests/modules/picard/fastqtosam/nextflow.config

```diff
@@ -0,0 +1,7 @@
+process {
+
+    publishDir = { "${params.outdir}/${task.process.tokenize(':')[-1].tokenize('_')[0].toLowerCase()}" }
+    withName: ".*:PICARD_FQTS_CUSTOM_SAMPLENAME" {
```

Comment:
I see why you used an import alias now. Could prefix with the workflow name to achieve the same thing, I guess.